### PR TITLE
[CustomerInsights] Update SDK code for swagger changes.

### PR DESCRIPTION
**Pull request by tjlvtao**
## Description

Update SDK:
1. 2 new APIs added.
2. Some model updates in swagger.

Swagger PR link:  https://github.com/Azure/azure-rest-api-specs/pull/1116

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [X] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [X] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [X] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [X] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as a link to the swagger spec, used to generate the code.
- [X] The `project.json` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/CustomerInsights/Management.CustomerInsights/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure CustomerInsights Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure CustomerInsights management functions for managing the Microsoft Azure CustomerInsights service.")]
 
-[assembly: AssemblyVersion("0.9.0.0")]
-[assembly: AssemblyFileVersion("0.1.0.0")]
+[assembly: AssemblyVersion("0.9.1.0")]
```
Comment (shahabhijeet) [inline src/SDKs/CustomerInsights/Management.CustomerInsights/Properties/AssemblyInfo.cs]: @tjlvtao why there is such huge difference between assembly and file version? Can this be made consistent.

File: src/SDKs/CustomerInsights/Management.CustomerInsights/Microsoft.Azure.Management.CustomerInsights.csproj [hunk 2]
```diff
@@ -2,7 +2,7 @@
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
   <PropertyGroup>
     <Description>Provides developers with libraries to create and manage Namespaces and manage Authorization Rules. Note: This client library is for CustomerInsights under Azure Resource Manager.</Description>
-    <VersionPrefix>0.1.0-preview</VersionPrefix>
+    <VersionPrefix>0.1.1-preview</VersionPrefix>
```
Comment (shahabhijeet) [inline src/SDKs/CustomerInsights/Management.CustomerInsights/Microsoft.Azure.Management.CustomerInsights.csproj]: @tjlvtao please remove all the other nuget package properties.
Only PackageId, AssemblyName, Description, PackageTags and versionPrefix stays from the first property group.
All the commented code should also be removed. This was kept for referencing old references, nuget references.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/CustomerInsights/Management.CustomerInsights/Microsoft.Azure.Management.CustomerInsights.csproj (hunk 2 above)
Comment (tjlvtao) [inline src/SDKs/CustomerInsights/Management.CustomerInsights/Microsoft.Azure.Management.CustomerInsights.csproj]: Done.

File: src/SDKs/CustomerInsights/Management.CustomerInsights/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (tjlvtao) [inline src/SDKs/CustomerInsights/Management.CustomerInsights/Properties/AssemblyInfo.cs]: Used 0.9.1.0 for both.

File: src/SDKs/CustomerInsights/Management.CustomerInsights/Microsoft.Azure.Management.CustomerInsights.csproj (hunk 2 above)
Comment (shahabhijeet) [inline src/SDKs/CustomerInsights/Management.CustomerInsights/Microsoft.Azure.Management.CustomerInsights.csproj]: @tjlvtao why AssemblyVersion is different from <VersionPrefix>, that is what I wanted you address and make it consistent.
I am not sure how your code got into this state, you cannot have nuget package say one version and assemblyversion be different. And if you still want to do this way, please let me know what can be your reasons to have such different versions between Assembly and nuget version.

**Comment by shahabhijeet** [comment]
@azuresdkci test this please

**Comment by shahabhijeet** [comment]
@azuresdkci test this please

**Comment by shahabhijeet** [comment]
@azuresdkci test this please

**Comment by shahabhijeet** [comment]
@azuresdkci test this please

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**